pdr_backend/lake/gql_data_factory.py

```diff
@@ -220,6 +220,8 @@ def _do_subgraph_fetch(
                     schema=dataclass.get_lake_schema(),
                 )
                 save_backoff_count = 0
+                if len(df["timestamp"]) == 0:
```

Comment:
could probably use the dict.get syntax, or at least some spacing from the previous and next rows.

Reply:
Can't use .get because that's a DataFrame not a dict